src/validation.cpp

```diff
@@ -730,8 +730,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     if (!bypass_limits && !CheckFeeRate(nSize, nModifiedFees, state)) return false;
 
     if (nAbsurdFee && nFees > nAbsurdFee)
-        return state.Invalid(TxValidationResult::TX_NOT_STANDARD,
-                "absurdly-high-fee", strprintf("%d > %d", nFees, nAbsurdFee));
+        LogPrintf("Ignoring Absurdfee\n");
```

Comment:
in commit 6a4a7c64246c826fed6b2bdf97ad91a5f864621b:

Any reason to change this? The next commit removes the line anyway.

Reply:
I wanted to make it really obvious that the behavior is identical if ATMP ignores high fees at this point, that's all :)